.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## Issue Name
+<!-- Whats this issue about? -->
+
+#### Issue Description
+<!-- Describe below when the issue happens and how to reproduce it -->
+
+
+#### Browser Information
```

Comment:
Change the heading to `System Configuration` or `System Information`. Because we aren't talking about browsers here